src/avram/uploadable.cr

```diff
@@ -0,0 +1,24 @@
+module Avram::Uploadable
```

Comment:
I was wondering, this serves both as an interface and a module. Should the interface be separated from the module?

Reply:
I've now separated them, so that's solved as well. :)